feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.ts

```diff
@@ -75,16 +75,6 @@ export class ConfiguratorCartEntryBundleInfoComponent {
       : false;
   }
 
-  /**
-   * Verifies whether the current screen size equals or is larger than breakpoint `BREAKPOINT.md`.
-   *
-   * @deprecated since 5.0 - method not used anymore
-   * @returns {Observable<boolean>} - If the given breakpoint equals or is larger than`BREAKPOINT.md` returns `true`, otherwise `false`.
-   */
-  isDesktop(): Observable<boolean> {
-    return this.breakpointService.isUp(BREAKPOINT.md);
-  }
-
```

Comment:
There should be a migration comment for this removal in docs/migration/6_0/migration-comments-members.json